ui/webui/test/check-basic

```diff
@@ -213,6 +213,28 @@ class TestBasic(anacondalib.VirtInstallMachineCase):
 
         self._testLogReview(b, m, "critical-error")
 
+    def testJsErrorHandling(self):
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+
+        i.open()
+
+        b.wait_not_present("#critical-error-bz-report-modal.pf-c-modal-box")
+
+        with self.assertRaises(Error):
+            b.eval_js("a.b()")
```

Comment:
I just realized you should only put the b.eval_js inside the with self.assertRaises. because otherwise all other errors forward in the test will not throw an exception.

Reply:
I am working on your test - I will push on your branch in a while.